src/dev/python/NativeCall.py

```diff
@@ -3,11 +3,11 @@
 from ctypes import *
 import os
 # for windows
-#dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
-#dll_helper = CDLL(os.getcwd() + '\dsn.dev.python_helper.dll')
+dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
```

Comment:
My bad. Using "platform.system()", that surely could work. I'll fix this in next pull request.